package.json

```diff
@@ -23,9 +23,9 @@
     "flow": "flow",
     "flow:check": "flow check",
     "test": "lerna run test",
-    "prepack:all": "for I in packages/*/; do echo \"cd $I && npm run prepack\" | perl -p -e 's/\\n/\\0/;'; done | xargs -0 node_modules/.bin/concurrently",
+    "prepack:all": "scripts/prepack-all",
```

Comment:
Order now matters (because TypeScript checks dependencies), so we're using a build script now.